packages/financial-templates-lib/src/clients/OptimisticOracleClient.ts

```diff
@@ -121,27 +121,31 @@ export class OptimisticOracleClient {
 
   // Returns an array of expired Price Proposals that can be settled and that involved
   // the caller as the proposer
-  public getSettleableProposals(caller: string): ProposePriceReturnValues[] {
+  public getSettleableProposals(callers?: string[]): ProposePriceReturnValues[] {
+    if (!callers) return this.expiredProposals;
+
     if (this.oracleType === OptimisticOracleType.SkinnyOptimisticOracle) {
       return (this.expiredProposals as SkinnyProposePrice["returnValues"][]).filter((event) => {
-        return ((event.request as unknown) as SkinnyRequest).proposer === caller;
+        return callers.includes(((event.request as unknown) as SkinnyRequest).proposer);
```

Comment:
I wish we did not have to do this two layered casting

Reply:
Yeah :/. I was hoping to clean this up, but I think it kinda has to be this way.